src/release/breaking-changes/navigator-and-page-api.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Navigator's page APIs breaking change
+description: >
+  Changes to Navigator's page APIs.
+---
+
+## Summary
+
+The [`Navigator`]'s page APIs are refactored so that they can integrate
+into Flutter's other pop mechanisms.
+
+## Context
+
+The `onPopPage` was added for cleaning up pages after a page is about to be popped.
+The developer can veto pop by returning false in the callback. This does not work well
+with other popping mechanisms in the framework, such as [`PopScope`] and cupertino back
```

Comment:
**question:** Is the cupertino mention an object or something else? Might be good to link to something.

Reply:
I'd change this even further:

Use the `onPopPage` method to clean up after a page is about to be popped.
You can veto the pop by returning `false` in the callback. This method doesn't work well
with other popping mechanisms in the framework,
such as [`PopScope`][] or [`CupertinoNavigationBarBackButton`][].

[`CupertinoNavigationBarBackButton`]: {{site.api}}/flutter/cupertino/CupertinoNavigationBarBackButton-class.html